roles/openshift_master/templates/master.yaml.v1.j2

```diff
@@ -102,7 +102,7 @@ masterPublicURL: {{ openshift.master.public_api_url }}
 networkConfig:
   clusterNetworkCIDR: {{ openshift.master.sdn_cluster_network_cidr }}
   hostSubnetLength: {{ openshift.master.sdn_host_subnet_length }}
-{% if openshift.common.use_openshift_sdn %}
+{% if openshift.common.use_openshift_sdn or openshift.common.use_nuage %}
   networkPluginName: {{ openshift.common.sdn_network_plugin_name }}
```

Comment:
@detiber I am specifying the network plugin name in my ansible hosts file as follows

os_sdn_network_plugin_name='nuage/vsp-k8s'